CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+
+## Contributing Changes
+
+Project maintainers will merge changes that both improve the project and meet our code quality standards. 
+
+### Suggested Workflow
+
+We recommend you to use the following workflow:
+
+1. Find an existing issue or create a new one.
+    - Seek feedback from the maintainers and the community that your proposed change is a good one.
+    - Clearly state whether you plan on implementing it yourself or not. If this is the case, you can request that the issue be assigned to you. Note: The person who filed the issue and the person who implements it don't have to be the same.
+2. Create a personal fork of the repository using GitHub (if you don't already have one).
+3. In your fork, create a branch based of main (`git checkout -b mybranch`).
+4. Create and commit your changes to your branch.
+    - Commit messages should follow the guidelines in [Commit Messages](#commit-messages)
+6. Run the linting and prettier commands with `pnpm lint` and `pnpm format` to ensure your code follows our code quality standards.
+7. Create a pull request (PR) against the compsci-adl/website repository's **main** branch.
+    - State in the description what issue or improvement your change is addressing.
+    - Check if all the Continuous Integration checks are passing. Refer to [Actions](https://github.com/compsci-adl/website/actions) to check for errors.
+8. Wait for feedback or approval of your changes from the maintainers
+9. When the maintainers have signed off, and all checks are green, your PR will be merged.
+    - You can safely delete the branch you used for making the change after the PR has been merged
+
+### Commit Messages
+Before you create a PR, please check whether your commits comply with the commit conventions used for this repository.
+When you create a commit, please follow the convention
+`<type>[optional scope]: <description>` in your commit message while using one of
+the following categories:
+
+- `feat`: Adds a new feature
+- `fix`: Fixes a bug (please reference an
+  existing issue if present)
+- `refactor`: Restructures code while not changing its original functionality
+- `docs`: Changing existing or creating new documentation (i.e.
+  README or other markdown files)
+- `build`: Changing the build system or dependencies
+- `style`: Related to code formatting (i.e.
+  Fixing linting or Prettier errors)
+- `ci`: Changing the configuration of continuous integration (i.e.
+  Github actions)
+- `chore`: Changes that do not fit into any of the above
+  categories
+
+If you are interested in the detailed specification you can visit
+<https://www.conventionalcommits.org/>
+
+### Code Style
+We follow the [AirBnb Javascript Style Guide](https://github.com/airbnb/javascript) and the [Airbnb React/JSX Style Guide](https://github.com/airbnb/javascript/tree/master/react). Please take the time to familiarise yourself with these guides.
```

Comment:
change "AirBnb" to "Airbnb"

Reply:
Actually we don't. I checked this style guide and ig it's too old for nextjs 14 and react 18.

Reply:
Yeah, I think I said we should somewhat follow the style guide but not completely.

Reply:
What would be the best way to word this?

Reply:
Maybe just remove this for now?